app/voting/Homepage/RoundActionCard/RoundActionCard.tsx

```diff
@@ -60,50 +71,59 @@ export const RoundActionCard = ({
           phaseEndsDaysFromToday !== 1 ? "s" : ""
         }`;
 
-  const labelContent = (() => {
-    const authedLabels: { [key in Phase]: string } = {
-      signups: `Next round starts ${specificDaysOrSoonLabel}`,
-      celebration: `Stay tuned for next round details!`,
-      voting: `Voting ends ${specificDaysOrSoonLabel}`,
-      covering: `Round ends ${specificDaysOrSoonLabel}`,
-    };
-
-    if (isAuthed) {
-      return authedLabels[phase];
-    } else {
-      return phase === "signups" ? (
-        <>{`Next round starts in ${phaseEndsDaysFromToday} days`}</>
-      ) : (
-        <>Notify me when next round starts</>
-      );
-    }
-  })();
-
-  const blurb = useBlurb({ phase, roundId, phaseEndsDatelabel });
+
+  // Do we need this???? 
```

Comment:
We should discuss it, but you can remove it for now. This is helper text, but I like the idea of things being so clear it's unnecessary. We can add something similar back later.

Reply:
Done and dusted